tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../src/tsconfig.json",
+    "compilerOptions": {
+      "baseUrl": "../src",
+    },
+    "include": [
+        "../src/indico/web/client/js/types",
+        "vc_zoom/indico_vc_zoom/client/**/*",
```

Comment:
Actually, this is prescient, since we don't have any Typescript in that dir (yet). I'll add it in my coming Zoom Rooms PR. Feel free to remove if it bothers you.